SQL/0000-00-00-schema.sql

```diff
@@ -2041,3 +2041,24 @@ CREATE TABLE `candidate_consent_history` (
   `EntryStaff` varchar(255) DEFAULT NULL,
   CONSTRAINT `PK_candidate_consent_history` PRIMARY KEY (`CandidateConsentHistoryID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE `visit` (
+  `VisitID` int(10) unsigned NOT NULL auto_increment,
+  `VisitName` varchar(100) NOT NULL,
```

Comment:
what name is that ? something like "V03" or "The 3rd visit" i.e. is this the new place for the visit_label or the front end label.

I think last discussion we said that we need both (one for presentation on the front end, the other to be used in file names and API ...)

Reply:
Actually no, I was ask to remove the front end label from the table as it was not need at this moment.

I can add it back @driusan if requested.

Reply:
i retract my comment then.

I understand that the current visitName is the the current visit_label